src/js/components/FormField/FormField.js

```diff
@@ -61,13 +72,24 @@ const isGrommetInput = (comp) =>
   (grommetInputNames.indexOf(comp.displayName) !== -1 ||
     grommetInputPadNames.indexOf(comp.displayName) !== -1);
 
+const getFocusStyle = (props) => {
+  if (
+    props.focus &&
+    props.shouldSkipFocus &&
+    props.theme.formField.focus.childFocus
```

Comment:
Do we need `?`s here
```suggestion
    props.theme.formField?.focus?.childFocus
```

Reply:
yes thought I had them think when I pulled out they got lost Thanks!